evidence/pool.go

```diff
@@ -306,6 +305,16 @@ func (evpool *Pool) SetLogger(l log.Logger) {
 	evpool.logger = l
 }
 
+// GetHeaderAtHeight gets the header from the block store at a specified height.
+// Is used for validation of LunaticValidatorEvidence
+func (evpool *Pool) GetHeaderAtHeight(height int64) *types.Header {
```

Comment:
```suggestion
func (evpool *Pool) Header(height int64) *types.Header {
```

https://golang.org/doc/effective_go.html#Getters